### PR TITLE
bump CDI version to 1.9.2

### PR DESCRIPTION
**Pull request by annastopel**
Bumps CDI vendored code to version 1.9.2

### CODE REVIEW

File: deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1160,7 +1160,7 @@ spec:
                   - hyperconverged-cluster-operator
                   env:
                   - name: OPERATOR_IMAGE
-                    value: rthallisey/hyperconverged-cluster-operator:latest
+                    value: docker.io/hyperconverged-cluster-operator:latest
```
Comment (djzager) [inline deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: Something isn't right here with this image value.

Comment (ohadlevy) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: its right with not using @rthallisey private one ;-)

**Comment by djzager** [comment]
[This is where](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/43ccbf50d4ba261b85a285ac99ed985fcbc7b9c1/pkg/components/components.go#L16-L18) the strange behavior comes from. 

I don't see anything in your PR that would cause this but the manifests need to be correct before merged. Maybe, you could:

1.  [modify this code](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/43ccbf50d4ba261b85a285ac99ed985fcbc7b9c1/pkg/components/components.go#L16-L18). This would make it so, from the manifest-templator you would have to be explicit where the image is coming from (which I think is probably better in the long run).
1. Update [the manifest-templator](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/43ccbf50d4ba261b85a285ac99ed985fcbc7b9c1/tools/manifest-templator/manifest-templator.go#L148-L152) to explicitly specify `docker.io/rthallisey` for the repository string. Once we start pushing these images to the kubevirt org...then we can update the manifests (and manifest-templator).

**Comment by djzager** [comment]
#111 fixes the issue. If you rebase and regenerate the manifests I believe the issue will have gone away.

**Comment by rthallisey** [comment]
@annastopel Does this need a backport to release-2.0?

**Comment by annastopel** [comment]
Yes, it fixes labels on cdi pods to cdi.kubevirt.io when deployed via olm

**Comment by rthallisey** [comment]
@annastopel can you file the backport?  Will review when the PR is up.

**Pull request merged**